Algorithms/PalindromeNumber.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Algorithms
+{
+    public class PalindromeNumber
+    {
+        /// <summary>
+        /// Complexity: O(N) - We use only one for, because the variant is to determine the midpoint
+        /// </summary>
+        /// <param name="number">value to evaluate</param>
+        /// <returns>true if is palindrome or false if is not</returns>
+        public bool IsPalindrome(int number) {
+            if (number < 0) return false; //Numbers negatives never will true
+            if (number <= 9 && number >= 0) return true; // Numbers with only digits always will true
+
+            int leng = number.ToString().Length;
+            char[] arr = number.ToString().ToCharArray();
+
+            int midpoint = (leng%2) == 1 ? (leng - 1) / 2 : leng / 2; // 1 is not even number || 0 is even number
+            for (int i = 0; i < midpoint; i++)
```

Comment:
good one! This is the best way to check for palindromes. 
You will see different things like reverse, or things like that but this way is "better", and actually this part could be O(n/2) since you are not iterating the whole array, you are iterating half of it